ansible_base/lib/dynamic_config/dynamic_settings.py

```diff
@@ -57,6 +57,11 @@
 except NameError:
     OAUTH2_PROVIDER = {}
 
+try:
+    TEMPLATES  # noqa: F821
+except NameError:
+    TEMPLATES = []
```

Comment:
Should we do the same thing for `FLAGS` (i.e. make sure it at least an ampty dict) or will the flags library work fine with everything above if its not defined?